apps/test/netsim/NetSimTable.js

```diff
@@ -394,6 +394,11 @@ describe("NetSimTable", function () {
     assertEqual(apiTable.log(), 'create[{}]');
   });
 
+  it ("calls multiCreate on the API table", function () {
+    netsimTable.multiCreate([{}, {}], callback);
+    assertEqual(apiTable.log(), 'create[{}]create[{}]');
```

Comment:
Nit: Might document expected order of operations better here by inserting non-identical objects.

```javascript
netsimTable.multiCreate([{x:1}, {x:2}], callback);
assertEqual(apiTable.log(), 'create[{x:1}]create[{x:2}]');
```